src/store/state/state-model.ts

```diff
@@ -128,9 +126,7 @@ export const stateModel: StateModelIF = {
       nameRequest: {
         legalType: ''
       },
-      nameTranslations: {
-        new: []
-      },
+      nameTranslations: [],
```

Comment:
Do we have to worry about loading existing IAs (with old schema)?

Reply:
@thorwolpert verified that in prod we don't have any IA with name translation. So no need to consider that